docs/testing.md

```diff
@@ -41,9 +41,9 @@ describe('your-app', () => {
   describe('your functionality', () => {
     it('performs an action', async () => {
       // Simulates delivery of a payload
-      // payload.event is the X-GitHub-Event header sent by GitHub (for example "push")
-      // payload.payload is the actual payload body
-      await app.receive(payload)
+      // event is the X-GitHub-Event header sent by GitHub (for example "push")
```

Comment:
Could you also update this from `event` to `context`, a name change from a while back.

Reply:
It looks like the this should still be called `event` on master https://github.com/probot/probot/blob/de18b9000b95667541d32c4d803300c2ae2c56da/src/application.ts#L200